common/src/KokkosFFT_utils.hpp

```diff
@@ -8,19 +8,60 @@
 #include <Kokkos_Core.hpp>
 #include <vector>
 #include <set>
+#include <string>
 #include <algorithm>
 #include <numeric>
 #include "KokkosFFT_traits.hpp"
 
+#ifndef KOKKOS_ENABLE_CXX17
```

Comment:
Testing C++20 features with a C++17 macro is not very future-proof.

You can define a `KOKKOSFFT_HAVE_C++20` macro built on top of the Kokkos one.

Reply:
I think it is OK to keep the same logic as `Kokkos`. After `Kokkos` drops C++17, I will just remove C++17, so it should be fine.